### ISSUE TITLE
`curious` not found

### ISSUE THREAD

**Issue by loganpowell** [pshrmn/curi#25]
I may be doing something wrong, but I'm on [Step 08](https://curi.js.org/tutorial/08-nav-react) of the tutorial and...

```
WARNING in ./src/components/AddToCart.js
105:15-22 "export 'curious' was not found in '@curi/react'
 @ ./src/components/AddToCart.js
 @ ./src/components/Book.js
 @ ./src/routes.js
 @ ./src/index.js
 @ multi (webpack)-dev-server/client?http://localhost:8080 ./index.js

WARNING in ./src/components/Checkout.js
79:15-22 "export 'curious' was not found in '@curi/react'
 @ ./src/components/Checkout.js
 @ ./src/routes.js
 @ ./src/index.js
 @ multi (webpack)-dev-server/client?http://localhost:8080 ./index.js
```

**Comment by loganpowell** [comment]
It looks like the `curious` HOC (used in the tutorial) used to be a regular React Component as shown in [the codesandbox](https://codesandbox.io/s/github/pshrmn/curi-tutorial/tree/09-now-what-react/?from-embed). Which should I use?

**Comment by pshrmn** [comment]
Sorry about that. You should use the `<Curious>` component now. I thought I had updated the tutorial, but I guess that I only updated the `curi-tutorial` repo. I will get that fixed and pushed soon.

**Comment by pshrmn** [comment]
Fixed in e54ed43cbfbd80d3f878087db5ab0ba1321e7d31
